resources/definitions/os_discovery/siteboss.yaml

```diff
@@ -2,6 +2,7 @@ mib: SITEBOSS-530-STD-MIB
 modules:
     os:
         sysDescr_regex: '/(?<hardware>SiteBoss \S+) (?<version>.*)/'
+        sysname: SITEBOSS-530-STD-MIB::siteName.0
```

Comment:
why is sysname not sysName?

Reply:
hi @murrant 

in https://github.com/librenms/librenms/pull/17972
we added `sysname:` option to YAML

should we change it to sysName ?

Reply:
private $osFields = [
        'version',
        'hardware',
        'features',
        'serial',
        'sysname',
    ];

Reply:
Yeah, I didn't see that in that PR :D

Reply:
> how about we fix it before merging?

as you wish, closed, changed, pushed all changes in one PR